cleansio/audio/audio_file.py

```diff
@@ -10,7 +10,7 @@ class AudioFile:
     def __init__(self, file_path):
         self.file_path = self.__converted_file_path(file_path)
         audio_segment = AudioSegment.from_file(self.file_path)
-        self.channels = audio_segment.channels
+        self.channels = self.convert_to_mono(audio_segment).channels
```

Comment:
Google Cloud Speech takes a file path. That file must be mono, what you're doing here is just creating an AudioSegment object that's mono.
Basically if the input file is not mono you have to loop through all the segment files that @levin-noro created and convert them into mono. Simply overwriting them is fine.